inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/broadcast.cpp

```diff
@@ -13,162 +13,161 @@ namespace {
 
 const std::vector<InferenceEngine::Precision> inputPrecisions = {
         InferenceEngine::Precision::FP32,
+        InferenceEngine::Precision::FP16,
         InferenceEngine::Precision::I32,
+        InferenceEngine::Precision::I16,
+        InferenceEngine::Precision::I8,
+        InferenceEngine::Precision::U8,
         InferenceEngine::Precision::BOOL
 };
 
-// NUMPY MODE
-
-std::vector<std::vector<size_t>> inShapesNumpy = {
-        {3, 1},
-        {1, 4, 1}
+// NUMPY MODE //////////////////////////////////////////
+// 1D
+std::vector<std::vector<size_t>> targetShapesNumpy1D = {
+        {1},
+        {16},
+        {1, 16},
+        {1, 1, 16},
+        {2, 16},
+        {2, 3, 6},
+        {1, 4, 4},
 };
 
-std::vector<std::vector<size_t>> targetShapesNumpy = {
+const auto numpyBroadcast1DInputParams = ::testing::Combine(
+        ::testing::ValuesIn(targetShapesNumpy1D),
+        ::testing::Values(ngraph::AxisSet{}), //not used in numpy mode
+        ::testing::Values(ngraph::op::BroadcastType::NUMPY),
+        ::testing::Values(std::vector<size_t>{1}),
+        ::testing::ValuesIn(inputPrecisions),
+        ::testing::Values(CommonTestUtils::DEVICE_CPU)
+);
+
+INSTANTIATE_TEST_CASE_P(TestNumpyBroadcast1D, BroadcastLayerTest, numpyBroadcast1DInputParams, BroadcastLayerTest::getTestCaseName);
```

Comment:
Please. add `smoke` prefix to add these tests to the pre-commit

Reply:
added